backend/kernelCI_app/views/logDownloaderView.py

```diff
@@ -58,6 +58,6 @@ def get(self, request):
 
         error_message = parsed_data.get("error")
         if error_message:
-            return JsonResponse({"error": error_message}, status=500)
+            return JsonResponse({"error": error_message}, status=400)
```

Comment:
would be a good ideia to use `getErrorResponseBody`

Reply:
Done